cognitedata/reveal
# feat(react-components): Add skybox and groundplane to use3dScenes

#### Type of change
![Feat](https://img.shields.io/badge/Type-Feat-green)  

## Description :pencil:
The `use3dScenes` hook should also export info about skybox and groundplane(s). This is needed in other to use the `use3dScenes` hook in another application. Now that `use3dScenes` and `useSceneConfig` basically returns the same, we should rewrite `SceneContainer` to use `use3dScenes` instead of `useSceneConfig`. That should however be done in a separate PR in my opinion.

## How has this been tested? :mag:
Have tested that the hook returns the correct data. 

## Test instructions :information_source:
To test open the `SceneContainer.stories` storybook and see that the SceneSelector button which depends on `use3dScenes` is still working


## Checklist :ballot_box_with_check:
